main/release/vars.py

```diff
@@ -12,3 +12,5 @@
 slack_token = os.environ.get('SLACK_API_TOKEN', 'no slack token in env')
 slack_client = WebClient(slack_token)
 slack_channel = os.environ.get('INPUT_SLACK_CHANNEL') or None
+
+dry_run = os.environ.get('DRY_RUN') or False
```

Comment:
`os.environ.get('DRY_RUN')` can be treated already in a boolean context so `or False` is redundant.